lib/rules/disallow-comma-before-line-break.js

```diff
@@ -51,12 +90,47 @@ module.exports.prototype = {
     },
 
     check: function(file, errors) {
+        var exceptFunction = this._exceptFunction;
+        var lineBreak = this._lineBreak;
+
+        function canSkip(token) {
+            if (!exceptFunction) {
+                return false;
+            }
+            var node = file.getNodeByRange(token.range[0]);
+
+            if (node.type !== 'ObjectExpression' ||
```

Comment:
What is this entire condition describing? Can you leave a comment with an example that would satisfy this condition?